src/xunit.analyzers/AssertEqualLiteralValueShouldBeFirst.cs

```diff
@@ -18,21 +19,31 @@ public class AssertEqualLiteralValueShouldBeFirst : AssertUsageAnalyzerBase
 
         protected override void Analyze(SyntaxNodeAnalysisContext context, InvocationExpressionSyntax invocation, IMethodSymbol method)
         {
-            if (invocation.ArgumentList.Arguments.Count < 2)
+            var arguments = invocation.ArgumentList.Arguments;
+            if (arguments.Count < 2)
                 return;
 
-            var firstArg = invocation.ArgumentList.Arguments[0];
-            var secondArg = invocation.ArgumentList.Arguments[1];
+            ArgumentSyntax expectedArg, actualArg;
+            if (arguments.All(x => x.NameColon != null))
+            {
+                expectedArg = arguments.Single(x => x.NameColon.Name.Identifier.Text == "expected");
```

Comment:
:bulb: Use `Identifier.ValueText` to ensure the alternate form `@expected` is handled as well.

Reply:
Thanks! I always get confused between those two, and usually, they're the same. Good to know a scenario where they differ. I'll fix.